### PR TITLE
AdminSearchPageUiTest failing #4188

### PR DESCRIPTION
**Pull request by taniach**
Fixes #4208

### CODE REVIEW

File: src/test/resources/pages/adminSearchPageDefault.html [hunk 1]
```diff
@@ -195,9 +195,9 @@
 
 
     <script src="/js/adminSearch.js" type="text/javascript"></script>
+    <script src="&#x2f;bootstrap&#x2f;js&#x2f;bootstrap.min.js" type="text&#x2f;javascript"></script>
```
Comment (wkurniawan07) [inline src/test/resources/pages/adminSearchPageDefault.html]: Why is this needed? Bootstrap should be loaded way up there in all pages.

**Comment by wkurniawan07** [comment]
Looks okay other than the one I pointed out. One thing I'm kind of curious though: why does `'` need to be replaced by `&quot;`?

File: src/test/resources/pages/adminSearchPageDefault.html (hunk 1 above)
Comment (taniach) [inline src/test/resources/pages/adminSearchPageDefault.html]: There was an extra Bootstrap include in the jsIncludes. I removed it.

**Comment by taniach** [comment]
I pushed the core prefs here by mistake
